src/bundle/commitments.rs

```diff
@@ -172,7 +172,7 @@ pub(crate) fn hash_issue_bundle_txid_data<A: IssueAuth>(bundle: &IssueBundle<A>)
             ind.update(note.rseed().as_bytes());
         }
         ia.update(ind.finalize().as_bytes());
-        ia.update(action.asset_desc().as_bytes());
+        ia.update(action.asset_desc().as_slice());
```

Comment:
why not just 
`ia.update(action.asset_desc());` ?

Reply:
also remove `.iter` from ` for action in bundle.actions() {` (line 165)

Reply:
Done.